ifopt_ipopt/src/ipopt_solver.cc

```diff
@@ -98,4 +107,16 @@ IpoptSolver::SetOption (const std::string& name, double value)
   ipopt_app_->Options()->SetNumericValue(name, value);
 }
 
+void
```

Comment:
All the lines below can be removed if I'm  not missing something?

Reply:
Removed.